index.js

```diff
@@ -25,11 +25,11 @@ module.exports = class MemeStealer extends Plugin {
         };
 
         const MessageContextMenu = await getModuleByDisplayName('MessageContextMenu');
-
-        const handleImageContext = function (args, res) {
+      
+        inject('meme-stealer-imageContext', MessageContextMenu.prototype, 'render', function (args, res) {
             const { target } = this.props;
       
-            if (target.tagName.toLowerCase() === 'img' && target.parentElement.classList.contains('pc-imageWrapper')) {
+            if (target.tagName.toLowerCase() === 'img' && target.parentElement.classList.contains('imageWrapper-2p5ogY')) {
```

Comment:
I would appreciate if you could get the CSS class using the `getModule` method from the webpack using the following code:
```js
const { imageWrapper } = getModule([ 'imageWrapper' ], false);
```
To get access to that just change 
```js
const { getModuleByDisplayName, React } = require('powercord/webpack');
```
to 
```js
const { getModuleByDisplayName, getModule, React } = require('powercord/webpack');
```
at the top of the index.js.

P.S.: I already had this fix locally but I forgot to push it and since you already made this PR, I'll let you fix it entirely so you didn't waste your time.